docs/src/configs/docs/competitions/endpoints.js

```diff
@@ -236,7 +236,7 @@ export default [
       },
       {
         description: 'If startsAt OR endsAt is a past date.',
-        body: { message: 'Invalid dates: All start and end dates must be in the future.' }
+        body: { message: 'Start date must be before the end date.' }
```

Comment:
Is the previous message not relevant anymore? This doesn't seem to be just a grammar correction.

Reply:
Yup, the invalid dates prefix was caused by the model validation, since most errors don't have such error prefix. I thought I'd remove it.

Reply:
No I mean the original meaning of the message, all start and end dates cannot be past dates?

Reply:
Yeah, sounds kinda weird, what'd you suggest? 🤔